pkg/controller/migplan/validation.go

```diff
@@ -114,6 +116,12 @@ func (r ReconcileMigPlan) validate(plan *migapi.MigPlan) error {
 		return err
 	}
 
+	// PV list.
+	err = r.validateClosed(plan)
```

Comment:
Fix cut-n-paste comment.

Perhaps the closed should be validated first and the remaining validations skipped (when closed).

Reply:
Sure. We could modify validateClosed to return a bool and an error, and if it returns `true`, we could return right there.